modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -45,4 +45,19 @@ spec:
 ----
 <1> By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.
 
+. Enable `devEnableEvictionsInBackground` to improve VM eviction stability during live migrations. This option allows evictions to occur in the background, mitigating issues with oscillatory behavior during live migration:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  devEnableEvictionsInBackground: true
+----
+
+This setting is useful when using the `LowNodeUtilization` profile to ensure more stable evictions.
```

Comment:
`LowNodeUtilization` isn't a "profile". It's a "strategy" that is technically part of the `LongLifecycle` profile.

So maybe this should say?

```suggestion
This setting is useful when using the `LongLifecycle` profile to ensure more stable evictions.
```